liferay-tango/liferay-portal
# LPS-129897 Allow select site and organization roles to create segments #1079

Motivation
=======
Right now it is not possible to select site or organization roles
(https://issues.liferay.com/browse/LPS-129897)

Proposed Solution
========
Add a new User Group Role selector (called "Site and Organization Role Selector") to allow users select this role. To do so, I had to create new ItemSelectorCriteria for this types and adapt current logic

The selector Role has been renamed as "Regular Role"

Screencaps
========

![Screenshot 2021-04-15 at 19 05 42](https://user-images.githubusercontent.com/5776822/114910053-b8681080-9e1d-11eb-8d1c-6bca4ec48951.png)

How to test it
========
1. Create a user and assign Site Role: Site Administrator
2. Create a Segment and add the Site and Organization Role Criterion.
3. Check that the number of members are correctly updated.

https://user-images.githubusercontent.com/5776822/114910666-74c1d680-9e1e-11eb-875c-647aaac2188f.mov
